buildcontext/git.go

```diff
@@ -235,6 +266,7 @@ func (gr *gitResolver) resolveGitProject(ctx context.Context, gwClient gwclient.
 					"git log -1 --format=%ae >/dest/git-author || touch /dest/git-author ; " +
 					"git log -1 --format=%b >/dest/git-body || touch /dest/git-body ; " +
 					"git for-each-ref --contains HEAD --format '%(refname:lstrip=-1)' >/dest/git-refs || touch /dest/git-refs ; " +
+					"find -name Earthfile > /dest/Earthfile-paths || touch /dest/Earthfile-paths ; " +
```

Comment:
Maybe add `-type f`, to ensure that they're files and not dirs?

Reply:
Sure.